.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file shouldn't be added to the commit. I thought the gitignore was ignoring these, not sure why this was able to be added!